File: cpp/grammer/14_counter.cpp
#include <iostream>
#include <string>
#include <unordered_map>

int main() {
    std::string text = "banana";
    std::unordered_map<char,int> counter;
    for (char c : text) counter[c]++;

    std::cout << "문자열: " << text << "\n";
    for (auto& [c, n] : counter) std::cout << c << ':' << n << ' ';
    std::cout << '\n';
}

File: javascript/grammer/14_counter.js
const text = 'banana';
const counter = {};

for (const ch of text) counter[ch] = (counter[ch] ?? 0) + 1;
const top1 = Object.entries(counter).sort((a, b) => b[1] - a[1])[0];

console.log('문자열:', text);
console.log('빈도수 객체:', counter);
console.log('가장 많이 나온 문자 1개:', top1);
